iac/terraform/aws/scripts.tf

```diff
@@ -0,0 +1,63 @@
+resource "null_resource" "run_update_methods_integrations_script" {
+  provisioner "local-exec" {
+    environment = {
+      OPENAPI_DIRECTORY = var.openapi_files_path
+      API_VERSION       = var.openapi_version
+      CLOUD_PROVIDER    = "aws"
+    }
+    command = "${path.module}/scripts/updateOpenApiIntegrations.sh"
+  }
+
+  depends_on = [
+    aws_api_gateway_rest_api.rest_api
```

Comment:
Does it depend on other scripts?

Reply:
no, this is the first script that runs after API gateway creation